src/components/global/NavBar/NavLInks.vue

```diff
@@ -2,65 +2,66 @@
 import { defineComponent } from 'vue';
 
 export default defineComponent({
-   name: 'NavLinks',
-    data() {
-        return {
-            urls: [
-                {id: 1, displayName: 'Movies', url: '#'},
-                {id: 2, displayName: 'Screenings', url: '#'},
-                {id: 3, displayName: 'Contact us', url: '#'},
-            ],
-        }
-    },
+  name: 'NavLinks',
+  data() {
+    return {
+      urls: [
```

Comment:
It looks like static (nonreactive) data so it also could be declared out of `defineComponent` options scope like `const urls = [...]`

Reply:
The problem is that when I move the `urls` array out of the `defineComponent` scope I cannot access it in the `<router-link />` `v-for` directive.

I'm not sure if this was what You had in mind, but I came across this type of solution on [dev.to](dev.to):
`export default defineComponent({
  name: 'NavLinks',
  urls: [
    { id: 1, displayName: 'Movies', routeName: 'Movies' },
    { id: 2, displayName: 'Screenings', routeName: 'Screenings' },
    { id: 3, displayName: 'Contact us', routeName: 'Contact' }
  ],
  computed: {
    useUrls() {
      return this.$options.urls;
    }
  }
});`

Reply:
you're right 👍  I forgot that template doesn't have access to values out of components scope. So `$options` is a good place for urls as you suggested 💪